mintcoin-qt.pro

```diff
@@ -123,9 +123,12 @@ contains(USE_BUILD_INFO, 1) {
     DEFINES += HAVE_BUILD_INFO
 }
 
-*-g++-32 {
```

Comment:
Oddly it seems that the old version of this code only used SSE2 instructions for 32-bit systems, not for 64-bit. That was broken, so this change is probably a big win for 64-bit systems as well as ARM. :smile: